presidio-analyzer/VERSION

```diff
@@ -0,0 +1 @@
+0.0.4
```

Comment:
0.0.1?

Reply:
Wanted it to be aligned with the next version of presidio

Reply:
Anyway this is just for internal purposes. We're not going to merge this for now.

Reply:
@eladiw changed to 0.0.1